fastlane/Fastfile

```diff
@@ -0,0 +1,69 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:android)
+
+platform :android do
+  desc "Runs all the tests"
+  lane :test do
+    gradle(task: "test")
+  end
+
+  lane :tests do
+    gradle(task: "test")
+  end
+
+  lane :upload_screenshots do
```

Comment:
Not clear if we're uploading to OONI or DW. I guess it would depend on the `play.json` file? Should we support different ones at the same time with the `json_key` argument?